rotary/repair.py

```diff
@@ -811,47 +854,50 @@ def main():
 
     dependency_dict = dict(zip(DEPENDENCY_NAMES, dependency_paths))
 
-    # Check output dir
-    try:
-        if os.path.isdir(output_dir):
-            logger.warning(f'Output directory already exists: "{output_dir}"; files may be overwritten.')
-
-        os.makedirs(output_dir, exist_ok=True)
-
-    except TypeError:
-
-        # Catches error that occurs if you run just 'repair.py' rather than 'repair.py -h' in the CLI
-        parser.print_help()
-        sys.exit(0)
+    # Parse some command line inputs further
+    assembly_info_type = 'custom' if args.custom_assembly_info_file is True else 'flye'
+    # TODO - improve error handling if a string is provided instead of a real length
+    length_thresholds = [int(x) for x in args.length_thresholds.split(',')]
+    cli_tool_settings_dict = {'flye_read_mode': args.flye_read_mode,
+                              'flye_read_error': args.flye_read_error,
+                              'circlator_min_id': args.circlator_min_id,
+                              'circlator_min_length': args.circlator_min_length,
+                              'circlator_ref_end': args.circlator_ref_end,
+                              'circlator_reassemble_end': args.circlator_reassemble_end}
+    threads_mem_mb = int(args.threads_mem * 1024)
 
     # Startup messages
     logger.info('Running ' + os.path.basename(sys.argv[0]))
-    logger.info('Version: ' + SCRIPT_VERSION)
     logger.info('### SETTINGS ###')
-    logger.info(f'Long read filepath: {long_read_filepath}')
-    logger.info(f'Assembly FastA filepath: {assembly_fasta_filepath}')
-    logger.info(f'Assembly info filepath: {assembly_info_filepath}')
+    logger.info(f'Long read filepath: {args.long_read_filepath}')
```

Comment:
A lot of these log statements could probably be moved to the debug level.

Reply:
Done